pkg/vpcmodel/explainabilityInput.go

```diff
@@ -306,8 +307,14 @@ func (c *VPCConfig) getNodesFromInputString(cidrOrName string) (nodes []Node,
 	if endpoint != nil {
 		return endpoint, noErr, nil
 	}
+	// 2. cidrOrName references subnet
+	subnetEndpoints := c.getNodesOfSubnet(cidrOrName)
+	if subnetEndpoints != nil {
```

Comment:
Is the case where a subnet has no VSIs handled well?

Reply:
depends how you define well ;-) but I guess the answer is no
The error message will be too general. I'll fix it.

Reply:
fixed, test added